src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.vb

```diff
@@ -0,0 +1,1084 @@
+    Initializer: IInvalidExpression (OperationKind.InvalidExpression, Type: ?, IsInvalid)
+  IVariableDeclaration: i2 As ? (OperationKind.VariableDeclaration, IsInvalid)
+    Initializer: IInvalidExpression (OperationKind.InvalidExpression, Type: ?, IsInvalid)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact(), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
+        Public Sub MixedDimAsNewAndEqualsDeclarations()
+            Dim source = <![CDATA[
+Module Program
+    Sub Main(args As String())
+        Dim i1, i2 As New Integer, b1 As Boolean = False'BIND:"Dim i1, i2 As New Integer, b1 As Boolean = False"
+    End Sub
+End Module
+    ]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (3 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: i1 As System.Int32 (OperationKind.VariableDeclaration)
+    Initializer: IObjectCreationExpression (Constructor: Sub System.Int32..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Int32)
+  IVariableDeclaration: i2 As System.Int32 (OperationKind.VariableDeclaration)
+    Initializer: IObjectCreationExpression (Constructor: Sub System.Int32..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Int32)
+  IVariableDeclaration: b1 As System.Boolean (OperationKind.VariableDeclaration)
+    Initializer: ILiteralExpression (Text: False) (OperationKind.LiteralExpression, Type: System.Boolean, Constant: False)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact(), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
+        Public Sub DimAsNewMultipleDeclarationsSameInitializerInstance()
+            Dim source = <![CDATA[
+Module Program
+    Sub Main(args As String())
+        Dim i1, i2 As New Integer'BIND:"Dim i1, i2 As New Integer"
+    End Sub
+End Module
+    ]]>.Value
+
+            Dim fileName = "a.vb"
+            Dim syntaxTree = Parse(source, fileName)
+            Dim compilation = CreateCompilationWithMscorlib(syntaxTree)
+
+            Dim node As SyntaxNode = CompilationUtils.FindBindingText(Of LocalDeclarationStatementSyntax)(compilation, fileName)
+            Assert.NotNull(node)
+
+            Dim tree = (From t In compilation.SyntaxTrees Where t.FilePath = fileName).Single()
+            Dim semanticModel = compilation.GetSemanticModel(tree)
+            Dim operation = CType(semanticModel.GetOperationInternal(node), IVariableDeclarationStatement)
+
+            Assert.Equal(2, operation.Variables.Count())
+            Dim var1 = operation.Variables(0)
+            Dim var2 = operation.Variables(1)
+
+            Assert.NotNull(var1.InitialValue)
+            Assert.NotNull(var2.InitialValue)
+
+            Assert.Same(var1.InitialValue, var2.InitialValue)
```

Comment:
>Assert.Same(var1.InitialValue, var2.InitialValue) [](start = 12, length = 49)

This is feels fine to me, but let's make sure design team is fine with this behavior.